it/assets/FinancialDetailsIntegrationTestConstants.scala

```diff
@@ -36,7 +36,7 @@ object FinancialDetailsIntegrationTestConstants {
       documentDescription,
       outstandingAmount = outstandingAmount,
       originalAmount = originalAmount,
-      documentDate = "2018-03-29"
+      documentDate = LocalDate.of(2018, 3, 29)
```

Comment:
Just a note: I would say that more readable would be `LocalDate.parse("2018-03-29")`.